how-it-works/3_governance/00_tokenomics.card.md

```diff
@@ -6,8 +6,6 @@ title: Tokenomics
 
 # Tokenomics
 
-The Internet Computer (IC) is underpinned by advanced tokenomics. 
-It makes use of an utility token called ICP, which is used for a variety of functions on the platform including participation in governance and paying for computation & storage. 
-In addition, the IC's tokenomic model provides a funding framework for exciting Web3 projects being built on the Internet Computer. 
+The Internet Computer makes use of an utility token called ICP. The ICP token is used for a variety of functions on the platform. Users can stake ICP giving them the right to vote and earn voting rewards paid in ICP. Investors can use ICP to invest in SNS sales launched on the Internet Computer, i.e., the initial offerings of the native DAOs. Developers use ICP token to purchase cycles to power their dapps on the Internet Computer. And node providers are renumerated for the compute power they provide to the Internet Computer platform. 
```

Comment:
I suggest to shorten the third sentence to "Users can stake ICP giving them the right to vote and earn voting rewards.", because (as Lara pointed out) rewards are paid out in maturity.

Reply:
Done